src/main/java/fish/payara/extras/diagnostics/collection/CollectorService.java

```diff
@@ -47,10 +47,7 @@
 import com.sun.enterprise.config.serverbeans.Domain;
 import com.sun.enterprise.config.serverbeans.Server;
 import fish.payara.enterprise.config.serverbeans.DeploymentGroup;
-import fish.payara.extras.diagnostics.collection.collectors.DomainXmlCollector;
-import fish.payara.extras.diagnostics.collection.collectors.HeapDumpCollector;
-import fish.payara.extras.diagnostics.collection.collectors.JVMCollector;
-import fish.payara.extras.diagnostics.collection.collectors.LogCollector;
+import fish.payara.extras.diagnostics.collection.collectors.*;
```

Comment:
please add specific imports instead of this one with the wildcard, is preferable to list each import